Cannot build tuple-keydef with Tarantool 2.10.3-0-g8aca8a1

**Environment:**
```
OS:           macOS
OS Version:   11.6.8
Architecture: amd64
```

**Tarantool:**
```
Tarantool 2.10.3-0-g8aca8a1
Target: Darwin-x86_64-Release
Build options: cmake . -DCMAKE_INSTALL_PREFIX=/usr/local/Cellar/tarantool/2.10.3_1 -DENABLE_BACKTRACE=ON
Compiler: /usr/bin/clang /usr/bin/clang++
C_FLAGS: -Wno-unknown-pragmas -fexceptions -funwind-tables -fno-common -msse2 -Wformat -Wformat-security -Werror=format-security -fstack-protector-strong -fPIC -std=c11 -Wall -Wextra -Wno-strict-aliasing -Wno-char-subscripts -Wno-gnu-alignof-expression
CXX_FLAGS: -Wno-unknown-pragmas -fexceptions -funwind-tables -fno-common -msse2 -Wformat -Wformat-security -Werror=format-security -fstack-protector-strong -fPIC -std=c++11 -Wall -Wextra -Wno-strict-aliasing -Wno-char-subscripts -Wno-invalid-offsetof -Wno-gnu-alignof-expression
```

**Steps to reproduce:**
```bash
cmake .
make
```

**Expected result:**
Build is successful.

**Actual result:**
```bash
[ 14%] Building C object extra/CMakeFiles/txt2c.dir/txt2c.c.o
[ 28%] Linking C executable txt2c
[ 28%] Built target txt2c
[ 42%] Generating postload.lua.c
[ 57%] Building C object tuple/CMakeFiles/keydef.dir/util.c.o
[ 71%] Building C object tuple/CMakeFiles/keydef.dir/keydef.c.o
/Users/y.lobankov/Workspace/tuple-keydef/tuple/keydef.c:78:33: error: implicitly declaring library function 'strlen' with type 'unsigned long (const char *)' [-Werror,-Wimplicit-function-declaration]
        if (luaL_loadbuffer(L, modsrc, strlen(modsrc), modfile) != 0)
                                       ^
/Users/y.lobankov/Workspace/tuple-keydef/tuple/keydef.c:78:33: note: include the header <string.h> or explicitly provide a declaration for 'strlen'
/Users/y.lobankov/Workspace/tuple-keydef/tuple/keydef.c:158:9: error: implicitly declaring library function 'strstr' with type 'char *(const char *, const char *)' [-Werror,-Wimplicit-function-declaration]
        return strstr(path, "[*]") != NULL;
               ^
/Users/y.lobankov/Workspace/tuple-keydef/tuple/keydef.c:158:9: note: include the header <string.h> or explicitly provide a declaration for 'strstr'
/Users/y.lobankov/Workspace/tuple-keydef/tuple/keydef.c:378:3: error: implicitly declaring library function 'memcpy' with type 'void *(void *, const void *, unsigned long)' [-Werror,-Wimplicit-function-declaration]
                memcpy(tmp, path, path_len + 1);
                ^
/Users/y.lobankov/Workspace/tuple-keydef/tuple/keydef.c:378:3: note: include the header <string.h> or explicitly provide a declaration for 'memcpy'
3 errors generated.
make[2]: *** [tuple/CMakeFiles/keydef.dir/build.make:99: tuple/CMakeFiles/keydef.dir/keydef.c.o] Error 1
make[1]: *** [CMakeFiles/Makefile2:160: tuple/CMakeFiles/keydef.dir/all] Error 2
make: *** [Makefile:146: all] Error 2
```

--- a/tuple/keydef.c
+++ b/tuple/keydef.c
@@ -31,6 +31,7 @@
 
 #include <assert.h>
 #include <stdint.h>
+#include <string.h>
 #include <lua.h>
 #include <lauxlib.h>
 #include <tarantool/module.h>

